3.analysis/scripts/nbconverted/2.gene_set_enrichment_analysis.py

```diff
@@ -114,38 +133,76 @@
 plt.title('Control Gene Set Enrichment Analysis')
 
 
-# In[10]:
+# In[11]:
 
 
 # Using VAE generated data
 
+pdf_path = "../1.data-exploration/figures/gsea_plots.pdf"
+pdf = matplotlib.backends.backend_pdf.PdfPages(pdf_path)
+
```

Comment:
Consider adding small documentation explaining this for loop.

Reply:
Will do